src/normalizations/normalize-soap-message-from-editor.ts

```diff
@@ -111,16 +111,68 @@ export const generateParticipantInformation = (resource: Editor): Array<Partial<
 		  );
 };
 
+const getOrganizer = ({
+	calendar,
+	sender,
+	organizer
+}: {
+	calendar?: CalendarEditor;
+	sender: CalendarSender;
+	organizer: CalendarOrganizer;
+}): { a?: string; d?: string; sentBy?: string } => {
```

Comment:
I found a little hard to understand which is the content of the returned properties, without reverse engineering the code. IMO it is better to choose more descriptive names for the properties

Reply:
As it was the function was correct because it was used only to create the organizer field of the soap message. Now it is used to make more things and its return properties should be renamed. Just remember to change how the organizer value at line (321-329) is passed to the body